netci.groovy

```diff
@@ -1134,83 +1188,102 @@ def static addTriggers(def job, def branch, def isPR, def architecture, def os,
             if (scenario == 'formatting') {
                 assert configuration == 'Checked'
                 if (os == 'Windows_NT' || os == 'Ubuntu') {
-                    Utilities.addGithubPRTriggerForBranch(job, branch, "${os} ${architecture} Innerloop Formatting")
+                    Utilities.addGithubPRTriggerForBranch(job, branch, "${os} ${architecture} Formatting")
                 }
+
                 break
             }
 
             switch (os) {
                 // OpenSUSE, Debian & RedHat get trigger phrases for pri 0 build, and pri 1 build & test
                 case 'Debian8.4':
                 case 'RHEL7.2':
-                    if (scenario == 'default') {
+                    if (isNormalOrInnerloop) {
```

Comment:
Won't this mean the same trigger phrase for both `innerloop` and `normal` jobs?

Also, the trigger phrase doesn't include `architecture` or `configuration`. Can't we fix that (namely, make sure all trigger phrases include os/architecture/configuration)? I generally (but not always) avoided changing what was already there, but I think for the new (innerloop) jobs, we should make them more verbose/complete.

Shouldn't the innerloop trigger phrases also include the word "innerloop"?

Reply:
This is a good catch thank you.